frontend/libs/studio-components/src/components/StudioRecommendedNextAction/context/StudioRecommendedNextActionContext.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { createContext, useMemo, useState } from 'react';
```

Comment:
Great job with the context! This is exactly what I meant by developing a reusable pattern. Well done! 👏